frontend/public/components/RBAC/bindings.jsx

```diff
@@ -270,16 +270,11 @@ const rowFilters = [
     type: 'role-binding-kind',
     selected: ['cluster', 'namespace'],
     reducer: bindingType,
-    items: ({ ClusterRoleBinding: data }) => {
-      const items = [
-        { id: 'namespace', title: 'Namespace Role Bindings' },
-        { id: 'system', title: 'System Role Bindings' },
-      ];
-      if (data && data.loaded && !data.loadError) {
-        items.unshift({ id: 'cluster', title: 'Cluster-wide Role Bindings' });
-      }
-      return items;
-    },
+    items: [
```

Comment:
```suggestion
   filterGroupName: Scope
    items: [
```
The name that would define this filter.